test/Interpreter/SDK/archive_attributes.swift

```diff
@@ -1,7 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-build-swift %s -module-name=test -DENCODE -o %t/encode
 // RUN: %target-build-swift %s -module-name=test -o %t/decode
-// RUN: %target-build-swift %s -module-name=test -Xfrontend -disable-llvm-optzns -emit-ir | %FileCheck -check-prefix=CHECK-IR %s
 // RUN: %target-run %t/encode %t/test.arc
 // RUN: plutil -p %t/test.arc | %FileCheck -check-prefix=CHECK-ARCHIVE %s
 // RUN: %target-run %t/decode %t/test.arc | %FileCheck %s
```

Comment:
This is an execution test; would it make sense to do it both the old way and the new way if we're on a new enough OS?

Reply:
(I'm not even 100% sure how to write such a test, but…)

Reply:
I can build the test with an old and new deployment target, and in the new deployment target case, add a runtime check that skips the test.